### DISCUSSION TITLE
My list of small additions I might plan to work on

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#53] [General]
Planned:

- TU reflections on windows
- Cyro engine optional patch for OMS engines
- Compatibility with habtech port and benjee10's APAS port.
- mod wiki pages :()
- possible patch for comfortable landing... 
- Mabye just Mabye I'll do a patch for something like kerbalism...
- waterfall configs for OMS

Completed patches: hullcam on reer doing port, mechjeb patch, and connected living space patch.

**Comment by FuryFire2004** [comment]
If any one wants to do these before I have time to feel free to!

**Comment by zer0Kerbal** [comment]
Transparent Pods would be a great one -- along with improved IVA - but both require Blender/Unity skills I do not possess.

**Reply by FuryFire2004** [reply]
i dont possess them ethier but mabye i can figure it out at some point

**Reply by zer0Kerbal** [reply]
> i dont possess them ethier but mabye i can figure it out at some point

fun skills to have. eventually I will bring my decades old CAD skills to bear and bite  the bullet and learn them as well. I am okay in GiMP/Photoshop and getting better all the time. Comes in handy for this work. :)

**Reply by zer0Kerbal** [reply]
@FuryFire2004 this is now done. Branch: Localization has been merged into 1.3.99.0-adopton branch, and have pulled changes from branch:master into 1.3.99.0-adopton branch.

Now for phase II of the part updating.

**Reply by FuryFire2004** [reply]
Yup

**Reply by zer0Kerbal** [reply]
> Now for phase II of the part updating.

FYI - Phase II is just going through the parts and fixing things that are way out of norm, along with adding @thumbs and [DRAG_CUBES], Phase III is for next release and includes Stock Cargo/Inventory for all parts, not just the cargo parts.

**Reply by zer0Kerbal** [reply]
Actually, having to go back and do Phase-IS (1.5) since there are still duplicate models/textures. If I can stop getting distracted by constant notifications from GitHub for nonsensical things

**Comment by zer0Kerbal** [comment]
@FuryFire2004 My plan today - after fixing my desk (part deux) is to finish the initial pass on the parts, then localize them and pull that branch into the current dev branch.

**Reply by FuryFire2004** [reply]
btw given you almost have a stable release done do you think we are getting close to realeasing this?

**Reply by zer0Kerbal** [reply]
> btw given you almost have a stable release done do you think we are getting close to realeasing this?

not.even.close. maybe in a week or two because I have other mods that are as pressing and issues in real life.

**Comment by zer0Kerbal** [comment]
also check out https://zer0kerbal.github.io/Antennas/ for wiki ideas. especially the PartInvoice

I've been using github pages instead of wiki because of automation that creates index pages nearly automatically.

**Reply by zer0Kerbal** [reply]
also - you probably know that you can clone / commit from a local repo for any wiki?
https://github.com/zer0Kerbal/Dreamer.wiki.git

**Reply by FuryFire2004** [reply]
huh didn't think of that

**Reply by zer0Kerbal** [reply]
> huh didn't think of that

certainly makes things easier to work locally - plus can use local links instead of url's.
[Typora](https://typora.io/) and VS Code are great tools for this. (both are free, (ish))

**Reply by FuryFire2004** [reply]
I'm waiting on the mod to get to a decent enough state for the build guide and glide profile bits, along side images so i dont cause people to drop dead at that one part we wont speak of... anyways yea i can start using those ig

**Reply by zer0Kerbal** [reply]
> I'm waiting on the mod to get to a decent enough state for the build guide and glide profile bits, along side images so i dont cause people to drop dead at that one part we wont speak of... anyways yea i can start using those ig

All good. the PartInvoice makes use of the autogenerated /Parts/@thumbs/ - so when they are available - well that's most of the work for that file. :D

**Reply by FuryFire2004** [reply]
I tend to prefer SS when i can because my visual mods are nice and  so they will be nice...

**Comment by FuryFire2004** [comment]
Eh the idea for an Iva quickly morphed into an idea for a independent mod which is based off crumb chaser yet is separate from this mod in its style and epic ness, if I'm to do the Iva it's gonna warrant me remodeling the whole thing and I am only doing that if I do major major changes to how the cream chaser replica is built, thus again, warranting that I should just develop a separate mod where I get that kind of u restricted creative freedom. I probably will be doing that honestly so it was nice modding with ya'll but I have my own independent original projects to pursue it appears :) I will check in on this project from time to time because I hope to see it come to life as well!

**Reply by FuryFire2004** [reply]
If you wanna know what happened to this post I made progress https://forum.kerbalspaceprogram.com/index.php?/topic/207373-dream-catcher-a-small-spaceplane/#comment-4105532 ig I was gonna name it cream chaser :( open to collaboration on a cream chaser mod tho one day ;)

**Reply by zer0Kerbal** [reply]
I wasn't saying no, rather not at this moment.
Still, Good Luck!

**Comment by FuryFire2004** [comment]
On Wed, Mar 23, 2022 at 12:35 AM zer0Kerbal ***@***.***>
wrote:

> I wasn't saying no, rather not at this moment.
> Still, Good Luck!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zer0Kerbal/Dreamer/discussions/53#discussioncomment-2419529>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHI7EU6GAWUBC6HVCZ5MHGDVBKNQVANCNFSM5RDR3OKQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
It’s not you it’s about a vision I have for it, I want to build from the
ground up over using an outdated framework

**Comment by AtomikkuSan** [comment]
Waterfall Configs!?
WHERE?
I'm pretty good at them, I've actually done a little bit of work for Stock Waterfall Effects!

**Reply by zer0Kerbal** [reply]
doesn't have them yet, and I have been pushing on other mods, will come back to dreamer shortly.
